util/meson.build

```diff
@@ -7,3 +7,14 @@ wlr_files += files(
 	'signal.c',
 	'time.c',
 )
+
+if uuid.found()
+	wlr_deps += uuid
+	add_project_arguments('-DHAS_LIBUUID', language: 'c')
```

Comment:
We try to avoid `#ifdef`s and use `#if` instead so that forgetting to define the macro triggers an error.